DEVELOP.md

```diff
@@ -107,78 +107,86 @@ Their upstream repos are assumed to have the remote name `origin`.
 
       Where the items in curly braces are replaced with their actual values.
 
-6.  Perform a complete build (in your favorite Python virtual environment):
+6.  Tag the version (temporary tagging, only locally):
 
-    - `make clobber all`
+    Note: This is the basis on which `pbr` determines the package version for
+    the subsequent steps. Because we also have PKG-INFO in the repo, tagging
+    now needs to be done in two steps, this temporary step and a subsequent
+    final step.
+
+    Create a tag for the new version (just locally):
+
+    - `git tag $MNU`
+
+    Remove the build products so that the next build picks up the tagged
+    version:
+
+    - `make clobber`
+
+7.  Perform a complete build (in your favorite Python virtual environment):
+
+    - `make all`
 
     If this fails, fix and iterate over this step until it succeeds.
 
-7.  Commit the changes and push to upstream:
+8.  Commit the changes and push to upstream:
+
+    Note: This also includes PKG-INFO in the commit, which now has the tagged
+    version.
 
     - `git status` - to double check which files have been changed
     - `git commit -asm "Release $MNU"`
     - `git push --set-upstream origin release_$MNU`
 
-8.  On Github, create a Pull Request for branch `release_$MNU`. This will
+9.  On Github, create a Pull Request for branch `release_$MNU`. This will
     trigger the CI runs in Travis and Appveyor.
 
     Important: When creating Pull Requests, GitHub by default targets
     the `master` branch. If you are releasing a fix version, you need to
     change the target branch of the Pull Request to `stable_M.N`.
 
-9.  Perform a complete test using Tox:
+10. Optional: Perform a complete test using Tox:
 
     - `tox`
 
     This will create virtual Python environments for all supported versions
     and will invoke `make test` (with its prerequisite make targets) in each
     of them.
 
-10. Optional: Perform a test in a local CI environment (Andy):
+11. Optional: Perform a test in a local CI environment (Andy):
```

Comment:
Question is a local test environment really a local CI environment??? or is it just a local test environment

Reply:
DONE - changed to "local multi-platform test environment"